.gitpod.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gitpod/workspace-full:2023-05-08-21-16-55
+
+# Some datasets work on 3.8 only
+RUN pyenv install 3.8.15\
+    && pyenv global 3.8.15
```

Comment:
From `kedro-datasets` 2.0.0 we've dropped python 3.8 support, so is this still needed?